config/pipeline.yaml

```diff
@@ -476,7 +476,51 @@ build_configs:
     branch: 'master'
     variants: *build-variants
 
-  stable-rc_6.6:
+  stable-rc_4.19: &stable-rc
     tree: stable-rc
-    branch: 'linux-6.6.y'
+    branch: 'linux-4.19.y'
     variants: *build-variants
+
+  stable-rc_5.4:
+    <<: *stable-rc
+    branch: 'linux-5.4.y'
+
+  stable-rc_5.10:
+    <<: *stable-rc
+    branch: 'linux-5.10.y'
+
+  stable-rc_5.14:
+    <<: *stable-rc
+    branch: 'linux-5.14.y'
+
+  stable-rc_5.15:
+    <<: *stable-rc
+    branch: 'linux-5.15.y'
+
+  stable-rc_5.16:
+    <<: *stable-rc
+    branch: 'linux-5.16.y'
+
+  stable-rc_5.17:
+    <<: *stable-rc
+    branch: 'linux-5.17.y'
+
+  stable-rc_5.18:
+    <<: *stable-rc
+    branch: 'linux-5.18.y'
+
+  stable-rc_5.19:
+    <<: *stable-rc
+    branch: 'linux-5.19.y'
+
+  stable-rc_6.1:
+    <<: *stable-rc
+    branch: 'linux-6.1.y'
+
+  stable-rc_6.3:
```

Comment:
This one is EOL and won't receive any more updates (same for 5.14, 5.16, 5.17, 5.18 & 5.19). It might be interesting to add 6.7 though, as this branch is still actively maintained.

Reply:
Updated, thanks!
6.3,  5.14, 5.16, 5.17, 5.18 & 5.19, removed, 6.7 added